README.md

```diff
@@ -268,7 +226,38 @@ See: http://dev.haufe.com/PostgreSQL-Cluster-Azure/
 All communication is over secure/encrypted channel
 (_by default at all times_) <br />
 to protect the data/privacy
-of people using the applications we make.
+of people using the applications we make. <br />
+
+We recommend using the "Let's Encrypt" service for SSL Certificates
+it's ***100% Free*** and (_provided by a Non-Profit foundation_) <br />
+to help you get started, we wrote a
+***step-by-step setup guide*** for apps deployed to Heroku:
+[SSL-certificate-step-by-step-setup-instructions.md](https://github.com/dwyl/learn-heroku/blob/master/SSL-certificate-step-by-step-setup-instructions.md)
+
+### Beginner Tutorials?
+
+We have _crafted_ a "***Complete Beginner's Guide***"
+for each element in the stack, so that we:
++ ***Document our collective learning***
```

Comment:
I don't know if this is the right place for this reasoning. I would suggest that our `start-here` repo needs to evolve into more of a contents directory for all our tutorials with the reasoning fitting in there. What do people reckon? https://github.com/dwyl/start-here/issues/64

For now, this makes sense 👍 :tada: